docs/examples/quick-start-with-terraform.md

```diff
@@ -0,0 +1,208 @@
+
+## Provision the infrastructure
+
+Before starting the tutorials below, you should ensure the following software is installed on your system:
+
+- [**Docker**](https://docker.io), the container runtime.
+- [**Kubectl**](https://kubernetes.io/docs/tasks/tools/install-kubectl/), the command-line tool for Kubernetes.
+- [**Terraform**](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli), the command-line tool for Terraform.
+
+Then, let's open a terminal on your machine and launch the following script, which creates the complete infrastructure with all playground already set up.
+
+{{ env.config.html_context.generate_clone_example_tf('quick-start-terraform') }}
+
+## Analyze the infrastructure and code
+
+With the previous command you created two KinD clusters, peered with each other. Furthermore, you offloaded a namespace in Liqo virtual cluster.\
+The choice of use of KinD was made because of there are no configuration requirements such as account creation or resources costs, therefore all the presented functionality works also on Cloud provider clusters.
+
+
+
+### KinD resource
+
+The first executed step by Terraform is the the creation of the KinD clusters: the resource in charge of building them is the `kind_cluster` resource of the provider [tehcyx](https://registry.terraform.io/providers/tehcyx/kind/latest/docs) which starting from configuration parameters (such as cluster_name, service_subnet/pod_subnet) generates the clusters and related config files needed by other providers to set up the infrastructure.
+
+You can inspect the deployed clusters by typing on your workstation:
+
+```bash
+kind get clusters
+```
+
+You should see a couple of entries:
+
+```text
+milan
+rome
+```
+
+This means that two KinD clusters are deployed and running on your host.
+
+Then, you can simply inspect the status of the clusters.
+To do so, you can export the `KUBECONFIG` variable to specify the identity file for _kubectl_ and _liqoctl_, and then contact the cluster.
+
+By default, the kubeconfigs of the two clusters are stored in the current directory (`./rome-config`, `./milan-config`).
+You can export the appropriate environment variables leveraged for the rest of the tutorial (i.e., `KUBECONFIG` and `KUBECONFIG_MILAN`), and referring to their location, through the following:
+
+```bash
+export KUBECONFIG="$PWD/rome-config"
+export KUBECONFIG_MILAN="$PWD/milan-config"
+```
+
+```{admonition} Note
+We suggest exporting the kubeconfig of the first cluster as default (i.e., `KUBECONFIG`), since it will be the entry point of the virtual cluster and you will mainly interact with it.
+```
+
+### Install Liqo with Helm resource
+
+After creating the clusters KinD Terraform will install Liqo using the resource `helm_release` of the provider [Helm](https://registry.terraform.io/providers/hashicorp/helm/latest/docs) once correctly configured with the cluster config files.
```

Comment:
```suggestion
After creating the two KinD clusters, Terraform will install Liqo using the `helm_release` resource of the [Helm provider](https://registry.terraform.io/providers/hashicorp/helm/latest/docs), once correctly configured with the cluster config files.
```
I do not understand exactly. With "once correctly configured with the cluster config files", do you mean:
- after having correctly configured the cluster config files
- (anything else?)

Reply:
We mean the correct configuration of the provider through config file or parameters.